csrc/ops/arith.cpp

```diff
@@ -2440,7 +2403,7 @@ TensorView* viewAsScalar(TensorView* inp) {
 
   IterDomain* id = IterDomainBuilder(
                        inp_domain[0]->container()->zeroVal(),
-                       IrBuilder::create<Int>(vec_size))
```

Comment:
I think this should be `int64_t` according to https://github.com/NVIDIA/Fuser/blob/6d5dfdb016f7c4e3e390d6d238adc1a0615e56cb/csrc/ir_interface_nodes.h#L74
and
https://github.com/NVIDIA/Fuser/blob/6d5dfdb016f7c4e3e390d6d238adc1a0615e56cb/csrc/ir_interface_nodes.h#L152